src/ontology/cl-edit.owl

```diff
@@ -32821,6 +32823,19 @@ SubClassOf(obo:CL_4052019 obo:CL_0000066)
 SubClassOf(obo:CL_4052019 ObjectSomeValuesFrom(obo:BFO_0000050 obo:UBERON_0003889))
 SubClassOf(obo:CL_4052019 ObjectSomeValuesFrom(obo:CL_4030045 obo:GO_0005929))
 
+# Class: obo:CL_4052020 (lung resident granzyme K-associated CD8 T cell)
+
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "doi:/10.1101/2022.06.17.496207") obo:IAO_0000115 obo:CL_4052020 "A granzyme K-associated CD8 T cell that resides in the lung, characterized by the expression of granzyme K (GZMK), and tissue residency markers CD103 and CD49a. This cell exhibits cytotoxic potential through its expression of multiple granzymes (GZMA, GZMB, GZMH, GZMM) in addition to GZMK.")
+AnnotationAssertion(terms:contributor obo:CL_4052020 <https://orcid.org/0000-0002-2336-2552>)
+AnnotationAssertion(terms:contributor obo:CL_4052020 <https://orcid.org/0009-0000-8480-9277>)
+AnnotationAssertion(terms:date obo:CL_4052020 "2024-10-14T15:42:41Z"^^xsd:dateTime)
+AnnotationAssertion(Annotation(oboInOwl:hasDbXref "PMID:28322863") Annotation(oboInOwl:hasDbXref "doi:/10.1101/2022.06.17.496207") terms:description obo:CL_4052020 "Lung resident granzyme K-associated CD8 T cell represents an intermediate stage of T cell differentiation, with GZMK expression peaking during this phase.")
```

Comment:
I might be wrong because I haven't use this AP, but I think this should be rdfs:comment and if this cell gets a definition by CELLxGENE it would be included here.